README.rst

```diff
@@ -134,6 +134,30 @@ If you don't have tox, you can generate a pex of tox:
 
     $ pex tox -c tox -o ~/bin/tox
 
+Tox provides many useful commands and options, explained at https://testrun.org/tox/en/latest/.
+Below, we provide some of the most commonly used commands used when working on Pex, but the
+docs are worth acquainting yourself with to better understand how Tox works and how to do more
+advanced commmands.
+
+To run a specific environment, identify the name of the environment you'd like to invoke by
+running ``tox --listenvs-all``, then invoke like this:
+
+.. code-block::
+
+    $ tox -e style
+
+All of our tox environments allow passthrough arguments, which can be helpful to run specific
```

Comment:
This is definitely not true. I think you mean to say "all of our to test environments"

Reply:
Auditing `tox.ini`, it seemed every environment extends `testenv`. Either way, fixed to qualify this, especially in case anything changes in the future.

Thanks for your help in landing this PR!